### ISSUE TITLE
Author with two last names separated by hyphen

### ISSUE THREAD

**Issue by djaiswal** [mitchelloharawild/vitae#92]
I have co-authors with two last names, separated by a hyphen, and I did not get expected output from detailed_entries function.

For example, if I want to include any paper from Ursula  [ https://scholar.google.com/citations?user=wB7ClR8AAAAJ ] in my CV, I have to make following correction

```r
pubs$author <- str_replace_all(pubs$author,"Ruiz‐Vera" , "Ruiz--Vera") 
```
to make sure desirable output.

I guess that the same problem can occur in other arguments of detailed_entries.

**Comment by robjhyndman** [comment]
Please provide a minimal reproducible example of the problem. The following seems to work ok:

```
scholar::get_publications("wB7ClR8AAAAJ") %>% 
  detailed_entries(
    when = year,
    with = author,
    what = title,
    where = journal,
)
```

**Comment by masumistadler** [comment]
If I may add to this, I had a similar problem with a hyphen in my publication title.
After knitting, the hyphen is a crossed box in the PDF. I'm not using `detailed_entries` but `bibliography_entries`. I did a quick search, it seems like an LaTeX issue. According to this [post](https://tex.stackexchange.com/questions/290794/how-to-deal-with-hyphens-in-bib-files), I suspect that `scholar` is using `‐` (U+2010) and LaTeX can only deal with `-` (U+0045). So, replacing the hyphen with `mutate_all(str_replace, pattern = "‐", replacement = "-")` fixed the issue for me.

```
scholar::get_publications("WDlaK4IAAAAJ&hl") %>% 
  distinct(title, .keep_all = TRUE) %>%
  filter(!str_detect(journal, "Abstract")) %>%
  transmute(bibtype = "Article", author = as.character(author),
            title = as.character(title),
            journaltitle = as.character(journal), year) %>%
  arrange(-year) %>%
  mutate(key = row_number()) %>%
  mutate_all(str_replace, pattern = "‐", replacement = "-") %>%
  RefManageR::as.BibEntry() %>% 
  RefManageR::WriteBib("stadler.bib")

bibliography_entries("stadler.bib")
```

**Comment by mitchelloharawild** [comment]
I'm hoping that this will get fixed via #112

**Comment by mitchelloharawild** [comment]
Closing as this appears to be an issue with using `{scholar}` and the format of the hyphens provided by the package. There may be circumstances in which the user wishes to use `-` (U+2010), and so vitae cannot always replace these characters - it is up to the user to make the modifications as described above.
